### PR TITLE
deleted variables from RootActivity, add-on style for text, fix bugs

### PR DESCRIPTION
**Pull request by GoldAlbatross**

### CODE REVIEW

File: app/src/main/java/com/example/shorts/ui/RootActivity.kt [hunk 1]
```diff
@@ -23,22 +26,20 @@ class RootActivity : ComponentActivity() {
...
+        this.window.statusBarColor = light.toArgb()
         super.onCreate(savedInstanceState)
         setContent {
             ShortsTheme {
-                val timeBox = viewModel.timeBox
-                val showStopButton = viewModel.stopButtonState
-                val time = viewModel.recoverTime
-                val checkIconState = viewModel.checkIconState
-                val startPhrase = viewModel.startPhrase
+                //val timeBox = viewModel.timeBox
```
Comment (OlegKostinDS) [inline app/src/main/java/com/example/shorts/ui/RootActivity.kt:34]: unnecessary comment

File: app/src/main/java/com/example/shorts/ui/theme/Type.kt [hunk 2]
```diff
@@ -11,9 +12,20 @@ import com.example.shorts.R
...
+        fontSize = 20.sp,
+        fontFamily = FontFamily(Font(R.font.kanit_cyrillic))
+    ),
+    caption = TextStyle(
+        color = Color.DarkGray,
+        fontSize = 12.sp,
+        fontFamily = FontFamily(Font(R.font.kanit_cyrillic))
     )
     /* Other default text styles to override
     button = TextStyle(
```
Comment (MargaritaBespalova) [inline app/src/main/java/com/example/shorts/ui/theme/Type.kt:31]: unnessassary comment

File: app/src/main/java/com/example/shorts/ui/widgets/StartButtonView.kt [hunk 3]
```diff
@@ -48,43 +44,53 @@ fun StartButton(
...
                     Text(
-                        text = startPhrase,
+                        text = viewModel.startPhrase,
                         modifier = Modifier.widthIn(max = 200.dp),
                         style = TextStyle.Default,
-                        fontSize = if (startPhrase.length > 14) 19.sp else 22.sp,
-                        fontFamily = best_bold,
+                        fontSize = if (viewModel.startPhrase.length > 14) 19.sp else 22.sp,
+                        fontFamily = cyrillic,
                         maxLines = 1,
```
Comment (MargaritaBespalova) [inline app/src/main/java/com/example/shorts/ui/widgets/StartButtonView.kt:81]: `fun lfkfkflsfk`

**Review by MargaritaBespalova — COMMENTED** [review]
все ок

File: app/src/main/java/com/example/shorts/ui/RootActivity.kt (hunk 1 above)
Comment (GoldAlbatross) [reply app/src/main/java/com/example/shorts/ui/RootActivity.kt:34]: все исправил

**Pull request merged**